content/departments/engineering/teams/code-intelligence/glossary.md

```diff
@@ -67,6 +107,16 @@ Examples:
 
 Many languages have only one (major) compiler. In such a situation, the compiler is sometimes referred to as "the (Language) compiler" instead of by its actual name.
 
+### cross-repository code navigation
+
+[Code navigation](#code-navigation) that works across different repositories. For example, some customers store their code across hundreds or even thousands of repositories. Cross-repository code navigation enables such customers to navigate their code as seamlessly as if it were stored in a single repository.
+
+Adding support for cross-repository navigation for a language requires adding the corresponding logic to the language's [indexer](#indexer). The [Sourcegraph docs on indexers](https://docs.sourcegraph.com/code_navigation/references/indexers) cover which indexers support cross-repository code navigation.
+
+### definition
+
+TODO
```

Comment:
Were you going to fill this in too or is it left as a future TODO?

Reply:
Future TODO.